.github/workflows/kind.yml

```diff
@@ -195,18 +197,32 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
+      - name: Download Spark jobs images from previous jobs
+        uses: actions/download-artifact@v3
+        with:
+          name: spark-jobs
       - name: Download ClickHouse monitor images from previous jobs
         uses: actions/download-artifact@v3
         with:
           name: clickhouse-monitor
-      - name: Load Theia image
-        run:  |
-          docker load -i clickhouse-monitor.tar
-          docker tag antrea/theia-clickhouse-monitor:latest projects.registry.vmware.com/antrea/theia-clickhouse-monitor:latest
       - name: Download ClickHouse server images from previous jobs
         uses: actions/download-artifact@v3
         with:
           name: clickhouse-server
+      - name: Download Theia Manager images from previous jobs
+        uses: actions/download-artifact@v3
+        with:
+          name: theia-manager
+      - name: Load Theia image
+        run:  |
+          docker load -i spark-jobs.tar
+          docker tag antrea/theia-spark-jobs:latest projects.registry.vmware.com/antrea/theia-spark-jobs:latest
+          docker load -i clickhouse-monitor.tar
+          docker tag antrea/theia-clickhouse-monitor:latest projects.registry.vmware.com/antrea/theia-clickhouse-monitor:latest
+          docker load -i clickhouse-server.tar
+          docker tag antrea/theia-clickhouse-server:latest projects.registry.vmware.com/antrea/theia-clickhouse-server:latest
+          docker load -i theia-manager.tar
+          docker tag antrea/theia-manager:latest projects.registry.vmware.com/antrea/theia-manager:latest
       - name: Load Theia image
         run:  |
           docker load -i clickhouse-server.tar
```

Comment:
Maybe remove this part as we already load the clickhouse-server in `Download Theia Manager images from previous jobs`